cluster/ci/config/prow/openshift/ci-operator/roles.yaml

```diff
@@ -660,3 +660,37 @@ items:
   - kind: ServiceAccount
     name: ci-operator
     namespace: ci
+
+- kind: Role
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: image-tagger
+    namespace: openshift-acme
+  rules:
+  - apiGroups:
+    - ""
+    - image.openshift.io
+    attributeRestrictions: null
+    resources:
+    - imagestreamtags
+    - imagestreams
+    - imagestreamimports
+    verbs:
+    - update
+    - patch
+    - get
+    - create
+
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: image-tagger-ci-operator
+    namespace: openshift-acme
+  roleRef:
+    kind: Role
+    name: image-tagger
+    namespace: openshift-acme
+  subjects:
+  - kind: ServiceAccount
+    name: ci-operator
```

Comment:
@petr-muller is this updated automatically? I am seeing
```
could not run post step : could not retrieve target imagestream: imagestreams.image.openshift.io "0.9" is forbidden: User "system:serviceaccount:ci:ci-operator" cannot get imagestreams.image.openshift.io in the namespace "openshift-acme": no RBAC policy matched
```
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/logs/branch-ci-tnozicka-openshift-acme-master-images/39

Reply:
Oh right, this one still isn't. I just applied it.